src/features/network-modification-table/use-modifications-selection.ts

```
@@ -124,11 +126,39 @@ export function useModificationsSelection({
     // after selecting the composite.
     useEffect(() => {
         setRowSelection((prev) => propagateSelectionToLoadedDescendants(prev, modifications));
     }, [modifications]);
 
-    return { rowSelection, onRowSelectionChange, lastClickedRowId, emitSelection };
+    return {
+        rowSelection,
+        onRowSelectionChange,
+        lastClickedRowId,
+        emitSelection,
+        getAllDescendants: getAllDescendants,
```

ESLint

[error] Expected property shorthand.

(object-shorthand)